src/Cache/_ScsControlClient.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Momento\Cache;
```

Comment:
nit, for consideration in later PRs: does PHP give class names that start with an `_` special treatment?  (e.g. does it hide them from autocomplete in IDEs?)  if not, maybe we move these to a sub-namespace.

Reply:
Let's talk about this offline . . . Not sure I completely understand what you're getting at.

Reply:
sure we can chat any time today